ClosedXML/Excel/ConditionalFormats/IXLConditionalFormat.cs

```diff
@@ -99,6 +100,7 @@ public interface IXLConditionalFormat
 
         IXLCFColorScaleMin ColorScale();
         IXLCFDataBarMin DataBar(XLColor color, Boolean showBarOnly = false);
+        IXLCFDataBarMin DataBar(XLColor color, XLColor colorNegative, Boolean showBarOnly = false);
```

Comment:
Please rename the variables to `positiveColor` and `negativeColor`.

Reply:
I change the names